packages/ibm-products/src/global/js/hooks/useRetrieveStepData.js

```diff
@@ -13,11 +13,11 @@ import { useEffect } from 'react';
  * @param {object} useResetCreateComponent
  * @param {object} useResetCreateComponent.stepsContext
  * @param {number} useResetCreateComponent.stepNumber
- * @param {boolean} useResetCreateComponent.introStep
- * @param {boolean} useResetCreateComponent.invalid
- * @param {boolean} useResetCreateComponent.shouldIncludeStep
- * @param {string} useResetCreateComponent.secondaryLabel
- * @param {string} useResetCreateComponent.title
+ * @param {boolean | undefined} useResetCreateComponent.introStep
+ * @param {boolean | undefined} useResetCreateComponent.invalid
+ * @param {boolean | undefined} useResetCreateComponent.shouldIncludeStep
+ * @param {string | undefined } useResetCreateComponent.secondaryLabel
+ * @param {string | node} useResetCreateComponent.title
  */
```

Comment:
@makafsal Due to `CreateFullPageStep.tsx` having these props be optional, when they are passed into this function they can be considered either their type, or undefined, if they are not passed in. However this function "requires" the props to not be undefined to "work", but in reality there are multiple checks that include the required props anyway. To not have to refactor this file within this same PR, this is a temporary solution to the problem.

Reply:
Got it, approving the PR from my end.